lib/Sema/ConstraintSystem.cpp

```diff
@@ -3741,21 +3741,13 @@ ConstraintSystem::getFunctionArgApplyInfo(ConstraintLocator *locator) {
     choice = overload->choice;
     rawFnType = overload->openedType;
   } else {
+    assert(isa<CallExpr>(anchor) && "Should always be a CallExpr");
```

Comment:
I don’t think this is really necessary as the `cast` below will assert it.

Reply:
Fixed, Thank you :)